srv/src/main/java/sap/capire/xtravels/handler/RecalculatePriceHandler.java

```diff
@@ -67,6 +71,20 @@ private void updateTotals(CqnStructuredTypeRef ref) {
         Update.entity(travel).data(Travels.TOTAL_PRICE, totalPrice).hint("@readonly", false));
   }
 
+    @After(event = EVENT_CREATE, entity = Travels_.CDS_NAME)
```

Comment:
```suggestion
    @After(event = EVENT_CREATE)
```
Not required as entity can be derived from the method signature.